micro-rdk/src/common/robot.rs

```diff
@@ -446,6 +504,17 @@ impl LocalRobot {
         Ok(res)
     }
 
+    pub fn get_periodic_app_client_tasks(&mut self) -> Vec<Box<dyn PeriodicAppClientTask>> {
+        let mut tasks = Vec::<Box<dyn PeriodicAppClientTask>>::new();
+
+        #[cfg(feature = "data")]
+        if let Some(dm_sync_task) = self.data_manager_sync_task.take() {
+            tasks.push(dm_sync_task);
+        }
+
+        tasks
```

Comment:
will we be moving the restart and config-change monitors from the entrypoints here as well?

Reply:
I wasn't planning to, but we could. I had to move the data manager ones because they need to exist on a robot created before we have network access. The restart monitor and config changes monitors do not have that constraint.

Reply:
@mattjperez - OK to resolve?